### DISCUSSION TITLE
Checking availability of current mirror: bad

### DISCUSSION THREAD

**Discussion by Craglitch** [termux/termux-packages#7850] [Q&A] [answered]
pkg install (package) or pkg update and pkg upgrade output :

Checking availability of current mirror: bad
Testing the available mirrors:
[*] https://packages.termux.org/apt/termux-main: bad
[*] https://deb.kcubeterm.me/termux-main: bad
[*] https://termux.mentality.rip/termux-main: bad
[*] https://grimler.se/termux-packages-24: bad
[*] https://termux.librehat.com/apt/termux-main: bad
Using fallback mirror: https://packages.termux.org/apt/termux-main
Ign:1 https://packages.termux.org/apt/termux-main stable InRelease
Ign:2 https://packages.termux.org/apt/termux-games games InRelease
Ign:3 https://packages.termux.org/apt/termux-science science InRelease
Ign:4 https://packages.termux.org/apt/termux-unstable unstable InRelease
Ign:1 https://packages.termux.org/apt/termux-main stable InRelease
Ign:2 https://packages.termux.org/apt/termux-games games InRelease
Ign:3 https://packages.termux.org/apt/termux-science science InRelease
Ign:4 https://packages.termux.org/apt/termux-unstable unstable InRelease
Ign:1 https://packages.termux.org/apt/termux-main stable InRelease
Ign:2 https://packages.termux.org/apt/termux-games games InRelease
Ign:3 https://packages.termux.org/apt/termux-science science InRelease
Ign:4 https://packages.termux.org/apt/termux-unstable unstable InRelease
Err:1 https://packages.termux.org/apt/termux-main stable InRelease
  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
Err:2 https://packages.termux.org/apt/termux-games games InRelease
  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
Err:3 https://packages.termux.org/apt/termux-science science InRelease
  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
Err:4 https://packages.termux.org/apt/termux-unstable unstable InRelease
  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: Failed to fetch https://packages.termux.org/apt/termux-main/dists/stable/InRelease  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
W: Failed to fetch https://packages.termux.org/apt/termux-games/dists/games/InRelease  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
W: Failed to fetch https://packages.termux.org/apt/termux-science/dists/science/InRelease  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
W: Failed to fetch https://packages.termux.org/apt/termux-unstable/dists/unstable/InRelease  Something wicked happened resolving 'packages.termux.org:https' (7 - No address associated with hostname)
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
The following packages were automatically installed and are no longer required:
  binutils libsasl
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

**Comment by sk0kanik** [comment, marked as answer]
Check your internet connection during pkg update and check any system setting preventing termux app from accessing internet.

**Reply by Craglitch** [reply]
Ok

**Comment by Smartsantosh1928** [comment]
Superb it's a only one answer
